crates/context/src/lib.rs

```diff
@@ -183,18 +196,6 @@ impl ContextManager {
             bail!("Application is not installed on node.")
         };
 
-        self.add_context(
-            &context,
-            identity_secret,
-            Some(ContextConfigParams {
-                protocol: self.client_config.new.protocol.as_str().into(),
-                network_id: self.client_config.new.network.as_str().into(),
-                contract_id: self.client_config.new.contract_id.as_str().into(),
-                application_revision: 0,
-                members_revision: 0,
-            }),
-        )?;
-
```

Comment:
when calling `init`, this "must've" been made available. so this can't be moved.

at this point, you can just make it empty.

Reply:
Reverted + proxy_contract empty string